backend/fittoring/src/main/java/fittoring/application/auth/presentation/AuthController.java

```diff
@@ -172,4 +177,17 @@ public ResponseEntity<LoginResponse> oauthSignUp(
         return ResponseEntity.status(HttpStatus.CREATED)
                 .body(response);
     }
+
+    @GetMapping("/find-login-id")
+    public ResponseEntity<LoginIdResponse> findLoginId(@RequestBody @Valid FindLoginIdRequest request) {
+        String loginId = authService.findLoginId(request.name(), request.phoneNumber());
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(new LoginIdResponse(loginId));
```

Comment:
p3: 사소하지만 이렇게도 할 수 있을 것 같아요! 
```suggestion
        return ResponseEntity.ok(new LoginIdResponse(loginId));
```
`ok()`는 내부적으로 `HttpStatus.OK`를 포함하고 있습니다!

<img width="516" height="168" alt="Image" src="https://github.com/user-attachments/assets/4e8fb152-7de2-4e51-a61c-3af8ce408cff" />

이렇게 하면 가독성이 조금! 더 올라갈 것 같네요 : )

Reply:
컨벤션을 통일하기 위해 status를 드러내고 있습니다!